_posts/en/pages/list/2016-03-10-announcements-join.md

```diff
@@ -9,7 +9,7 @@ version: 2
 ---
```

Comment:
Please increment the `version` field on line 8 so we can keep the translations in sync.